dev-python/pyside/pyside-9999.ebuild

```diff
@@ -135,8 +142,27 @@ src_configure() {
 
 	configuration() {
 		local mycmakeargs=(
+			-DPYTHON_EXECUTABLE="${PYTHON}"
```

Comment:
Ah, yes. I see. This is [issue 511](https://bugreports.qt.io/browse/PYSIDE-511) again, isn't it? The fix is trivial, so I hope for a speedy resolution.

We could temporarily resolve this on our end by also passing `-DPYTHON_LIBRARIES` and `-DPYTHON_INCLUDE_DIRS`. But, really. They just need to fix this.